docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller_test.go

```diff
@@ -0,0 +1,200 @@
+/*
+We can create the tests in a separate file, cronjob_controller_test.go.
```

Comment:
Ideally, we should have 1 file to the test each controller/reconciler. So, I'd suggest we shape it with something as:
 
```suggestion
Ideally, we should have one `<kind>_conroller_test.go` for each controller scaffolded and called in the `test_suite.go`. So, let's write our example test for the CronJob controller (`cronjob_controller_test.go.`)    
```